runtime/compiler/optimizer/TreeLowering.cpp

```diff
@@ -0,0 +1,444 @@
+
```

Comment:
Remove initial empty line

Reply:
Fixed in https://github.com/eclipse/openj9/pull/11968/commits/5ffd29207337aed764aca948f6754268061d30e8.